templates/netlify/remix.init/tsconfig.json

```diff
@@ -0,0 +1,20 @@
+{
```

Comment:
This file is the exact same as we have in the root of the template, so I don't think we need this one?
It's also not copied over in the `remix.init` script.